client/src/enums/nr-state.ts

```diff
@@ -12,5 +12,5 @@ export enum NrState {
   REFUND_REQUESTED = 'REFUND_REQUESTED',
   REFUNDED = 'REFUNDED', // NOT YET IMPLEMENTED
   REJECTED = 'REJECTED',
-  RESERVED = 'RESERVED',
+  RESERVED = 'RESERVED', // INTERIM STATE
```

Comment:
At a meeting today with Kaine, Janis and Lorna, it was said that COND_RESERVED and RESERVED are interim states that the UI will never see, however I noticed some UI pages that check for these states.

Perhaps it was meant that the "Existing Request Display" page will never see these (since the user will not be able to query the NR using its temp number) (the NR has only a temp number at this point).